lib/linguist/languages.yml

```diff
@@ -4486,6 +4486,16 @@ MiniD:
   tm_scope: none
   ace_mode: text
   language_id: 231
+MiniScript:
+  type: programming
+  color: "#4B4A56"
+  extensions:
+  - ".gs"
+  - ".ms"
+  - ".src"
```

Comment:
This is a very generic extension and as this is the only instance of it, the heuristic will not be used and every `.src` file will be classified as this language.

You could add it to the `generic.yml` file which would then use the heuristic, but then your heuristic needs to be 100% bomb proof with zero scope for error.

If in doubt, I recommend _not_ adding it.